app/controllers/event_controller.rb

```diff
@@ -0,0 +1,49 @@
+# main event applciation
+class EventController < ApplicationController
```

Comment:
[ ] **[OPTIONAL]** putting attending event functionality here breaks encapsulation because `events_controller` knows about two concerns "events" and "attendance". A better approach is to move creating attendance (or removing attendance) on its own controller `event_attendance_controller`. A good rule of thumb, if you find yourself having actions in your controller with names other than the default "RESTFUL" actions, your should probably separate those actions into another controller. In `event_attendance_controller.rb` you will have this method as `create` "creating attendance" which comply with the RESTUL naming. Read this article in more details [organizing controllers](http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/)